js/controller.js

```diff
@@ -1313,6 +1319,15 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.trySetAnamorphicFixState(true);
       // In case ad was skipped or errored while stalled
       this.setBufferingState(false);
+
+
+      var vrViewingDirectionList = [
```

Comment:
Could we add a more descriptive comment here? For someone without context it's hard to understand what this is meant to do. It might also make sense to move this to own its own function with a descriptive name

Reply:
\+